ws-nextjs-app/cypress/support/commands.ts

```diff
@@ -17,9 +17,42 @@ declare global {
         service: Services;
         pageType: PageTypes;
         variant?: Variants;
-        id: string;
+        id?: string;
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
       }) => Chainable<any>;
     }
   }
 }
+
+const getPageData = ({
+  service,
+  pageType,
+  variant,
+  id,
+}: {
+  service: Services;
+  pageType: PageTypes;
+  variant?: Variants;
+  id?: string;
+}) => {
+  const baseUrl =
+    Cypress.env('APP_ENV') === 'local'
+      ? 'http://localhost:7081'
+      : Cypress.config().baseUrl;
+  let path = `/${service}/${pageType}/${id}${variant ? `/${variant}` : ''}`;
+
+  if (service === 'ws') {
+    path = '/ws/languages';
+  }
+
+  return cy
+    .request({
+      url: `${baseUrl}${path}`,
+      failOnStatusCode: false,
+    })
+    .then(response => {
+      return response.body;
+    });
```

Comment:
I don't think we should do this if we can avoid it. I believe that there should be a script on the window which includes `__NEXT_DATA__` - this function should try to get at that data instead of fetching it using cy.request.